app/views/webauthn_credentials/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_for webauthn_credential, html: { class: "js-new-webauthn-credential--form" } do |f| %>
+  <div class="text_field">
+    <%= f.label :nickname, t('.nickname'), class: 'form__label' %>
+    <%= f.text_field :nickname, class: 'form__input js-new-webauthn-credential--nickname', required: true %>
+  </div>
+
+  <p hidden class="l-text-red-600 js-new-webauthn-credential--error"></p>
```

Comment:
> Would it be possible to rewrite the given error into something more user friendly?
Hmmm this error is returned in accordance with the W3C webauthn[ standard ](https://www.w3.org/TR/webauthn/). I'm a bit wary to overwrite it locally because the error type ("[InvalidStateError](https://webidl.spec.whatwg.org/#invalidstateerror)") is pretty generic and seems like it could be used for a wide variety of things. I don't want to switch based on an error type that might not always mean the same thing and accidentally end up obscuring other errors. Additionally, in chrome at least, the browser already returns a more user friendly message: 

<img width="811" alt="image" src="https://user-images.githubusercontent.com/8496209/189736286-4f69279d-fbc0-4739-9bbd-adc5dbf986ff.png">

What do you think?